tests/FileUpdates/TemporaryUpdateTest.php

```diff
@@ -603,7 +603,7 @@ public function foo() : void {
 
                             class A {
                                 public function foo() : void {
-                                    throw new Error("bad", 5);
+                                    throw new Error("bad", []);
```

Comment:
php-parser 5.0 only allows passing an array as the second constructor argument. php-parser 4.0 allows both.